src/tutors/assessments/edit/edit_assessment.php

```diff
@@ -500,7 +500,7 @@ function do_command(com) {
         <table class="form" cellpadding="2" cellspacing="2">
         <tr>
           <td><label for="feedback_title">Title &nbsp;</label></td>
-          <td><input type="text" name="feedback_title" value="<?php echo $assessment->feedback_name; ?>" size="40" maxlength="40"/></td>
+          <td><input type="text" name="feedback_title" value="<?php echo $assessment->feedback_name; ?>" size="40" maxlength="40"/>&nbsp;max 40 characters.</td>
```

Comment:
Can this field accept more characters than 40 in the DB? If so, could we increase the max length to match the DB constraint?

Reply:
done, DB limit is 45 chars.